### PR TITLE
ruby26.y: branch parser.

### PR DESCRIPTION
**Pull request by iliabylich**
Probably should be merged after 2.6.0-stable release.

### CODE REVIEW

File: lib/parser/current.rb [hunk 1]
```diff
@@ -65,10 +65,19 @@ def warn_syntax_deviation(feature, version)
     require 'parser/ruby25'
     CurrentRuby = Ruby25
 
+  when /^2\.6\./
+    current_version = '2.6.0'
```
Comment (whitequark) [inline lib/parser/current.rb]: If you change this to `current_version = '2.6.0-dev'`...

File: lib/parser/current.rb [hunk 2]
```diff
@@ -65,10 +65,19 @@ def warn_syntax_deviation(feature, version)
...
+
+    require 'parser/ruby26'
+    CurrentRuby = Ruby26
+
   else # :nocov:
     # Keep this in sync with released Ruby.
-    warn_syntax_deviation 'parser/ruby25', '2.5.x'
-    require 'parser/ruby25'
-    CurrentRuby = Ruby25
+    warn_syntax_deviation 'parser/ruby26', '2.6.x'
```
Comment (whitequark) [inline lib/parser/current.rb]: ... and revert this change, this PR can be merged right now, and other work landed on top of it. This is what we generally do.

**Comment by whitequark** [comment]
Also, branching a parser is not a user-visible feature, so it doesn't need a changelog entry.

**Comment by iliabylich** [comment]
@whitequark Done. Rebased on the latest master, included latest fix into 2.6 parser, applied changes per your comments.
